rdtools/filtering.py

```diff
@@ -31,13 +31,14 @@ def normalized_filter(normalized, low_cutoff=0.01, high_cutoff=None):
     return (normalized > low_cutoff) & (normalized < high_cutoff)
```

Comment:
`filtering.normalized_filter` currently has `normalized` as a parameter.  Change to `energy_normalized`?

Reply:
I do think that 'normalized' is a little vague...